com.liferay.blade.upgrade.liferay70/src/com/liferay/blade/upgrade/liferay70/apichanges/MavenPomLegacyDependencies.java

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.liferay.blade.upgrade.liferay70.apichanges;
+
+import aQute.lib.io.IO;
+
+import com.liferay.blade.api.AutoMigrateException;
+import com.liferay.blade.api.AutoMigrator;
+import com.liferay.blade.api.FileMigrator;
+import com.liferay.blade.api.Problem;
+import com.liferay.blade.api.SearchResult;
+import com.liferay.blade.api.XMLFile;
+import com.liferay.blade.upgrade.liferay70.XMLFileMigrator;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.wst.sse.core.StructuredModelManager;
+import org.eclipse.wst.sse.core.internal.provisional.IndexedRegion;
+import org.eclipse.wst.xml.core.internal.provisional.document.IDOMElement;
+import org.eclipse.wst.xml.core.internal.provisional.document.IDOMModel;
+import org.osgi.service.component.annotations.Component;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Text;
+
+@Component(
+	property = {
+		"file.extensions=xml",
+		"problem.title=maven pom legacy dependencies",
+		"problem.summary=maven pom legacy dependencies",
+		"problem.tickets=",
+		"problem.section=#maven-pom-legacy-dependencies",
+		"implName=MavenPomLegacyDependencies",
```

Comment:
No need to add this `implName` in the test, we can just create the filter for all that have auto.correct=maven-pom-legacy-dependencies and then search by `instanceof` classname.